zerver/openapi/js_examples.js

```diff
@@ -0,0 +1,68 @@
+const zulip = require('zulip-js');
+const fs = require('fs');
+
+const config = {
+    username: process.env.ZULIP_USERNAME,
+    apiKey: process.env.ZULIP_API_KEY,
+    realm: process.env.ZULIP_REALM,
+};
+
+// Object with an array of responses from the
+// code examples. It is written to 'js_example_responses.json'
+// later.
+const example_responses = {
+    responses: [],
+};
+
+const example_snippets = {};
+
+const examples = {};
+// Gets the responses from the code examples and
+// puts them in examples_output.responses .
+async function process_data() {
+    let response;
+    for (const method in examples) {
+        response = await examples[method](config, zulip);
+        const obj = {};
+        obj[method] = response;
+        example_responses.responses.push(obj);
+    }
+    return example_responses;
+}
+
+function generate_code_examples() {
+    for(let method in examples) {
+        example_snippets[method] = examples[method].toString();
+    }
+    fs.writeFileSync('./zerver/openapi/js_example_code.json', JSON.stringify(example_snippets, null, 4));
```

Comment:
From a consistency standpoint, let's have this return the data, and handle the printing/writing to file in the switch-case.